### PR TITLE
Update rubocop and setup travis

### PR DESCRIPTION
**Pull request by gbp**
As part of the Rubocop upgrade I've switched to using the safe navigation operator

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sudo: false
+language: ruby
+cache: bundler
+rvm:
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
+script:
+  - export RAILS_ENV=test
```
Comment (chrismytton) [inline .travis.yml]: I think the recommended way to define environment variable on Travis is using the `env:` key. The [Travis' Environment Variables docs](https://docs.travis-ci.com/user/environment-variables/#Defining-public-variables-in-.travis.yml) have more details.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  - bundle exec rake
```
Comment (chrismytton) [inline .travis.yml]: If you use the `env:` section as I've suggested above then I think you can get rid of this line as it's the default for `language: ruby`.

**Comment by gbp** [comment]
Normally I probably would done separate PRs but as they are vaguely related it would be fine just this once :)

**Comment by tmtmtmtm** [comment]
Is there a reason this actually needs to run on ruby 2.3? i.e. rather than unrolling things that were only introduced in 2.4, might it be simpler to say this is 2.4+? 

In general I'm in favour of supporting old versions in gems, so if we think that there's value in this, then I've no problem with it, but I'm curious as to whether there actually is. (I don't really know who the expected users of this are)

**Comment by gbp** [comment]
I was thinking there might be some of our projects where we're we're still using Ruby 2.3 either locally or in production. As it was a trivial change decided to go for it

**Pull request merged**